build.gradle.kts

```diff
@@ -5,18 +5,35 @@ import java.time.format.DateTimeFormatter
 plugins {
     `maven-publish`
     kotlin("jvm") version "1.4.0"
+    id("com.jfrog.bintray") version "1.8.5"
     id("org.ajoberstar.reckon") version "0.12.0"
 }
 
+val group = "com.chutneytesting"
+val projectName = "chutney-kotlin-dsl"
+val timestamp: String = DateTimeFormatter.ISO_INSTANT.format(Instant.now())
+val gitHubUrl = "https://github.com/chutney-testing/${projectName}"
+val publicationName = "default";
+
 reckon {
     scopeFromProp()
     snapshotFromProp()
 }
 
-group = "com.chutneytesting"
-
-val timestamp: String = DateTimeFormatter.ISO_INSTANT.format(Instant.now())
-val gitHubUrl = "https://github.com/chutney-testing/${project.name}"
+bintray {
+    user = System.getenv("BINTRAY_SERVER_USERNAME")
+    key = System.getenv("BINTRAY_SERVER_PASSWORD")
+    publish = true
+    setPublications(publicationName)
+    pkg(delegateClosureOf<com.jfrog.bintray.gradle.BintrayExtension.PackageConfig> {
```

Comment:
Here https://docs.gradle.org/current/userguide/kotlin_dsl.html#groovy_closures_from_kotlin, closureOf() is used. Have you experienced NPE to use the delegateClosureOf() method ?